container_files/nats/js.conf

```diff
@@ -0,0 +1,4 @@
+// change max payload from default 1MB to 8MB
```

Comment:
the comment here needs to change to 64MB for consistency (it was wrong when I sent it to you)

Reply:
done!